src/src_V2/modules/manager/user-management/index.jsx

```diff
@@ -0,0 +1,110 @@
+import { useEffect, useState } from 'react';
+import { Edit } from '@mui/icons-material';
+import { Routes, Route, useNavigate } from 'react-router-dom';
+import { Button, Select, IconButton, Box, MenuItem, Typography } from '@mui/material';
+
+import { Table } from '../../../components';
+import UserConfigContent from './UserConfig/UserConfig';
+import PageWrapper from '../../../components/userPageWrapper';
+
+import styles from './styles.module.css';
+import { mockUserManagementData } from './mock/data';
+import { useTableDataGenerator } from '../../../utils/hooks';
+import userManagementTableDataGenerator from './utils/table-data-generator';
+
+const EditBtn = ({ rowData, handleEdit }) => (
+  <span>
```

Comment:
what is the purpose of span? I think there will not be the difference if you remove it.

Reply:
thanks